happy-travel/edo
# Feature/aa 1510 add credit card price field to rate

Issue: https://github.com/happy-travel/agent-app-project/issues/1510
Insomnia: Edo -> Agents -> Availablity -> Start/Get result; Edo -> Agents -> Bookings -> Create Bookings(3 different payments)
Tested through insomnia and debug 

Test case:
  1) Run availablity search and check if through each step results roomSets contain new field CreditCardPrice
  2) Check if commission was appyied through creating booking with creditCard payment, otherwise no any commission
